time_tracker_api/time_entries/time_entries_namespace.py

```diff
@@ -256,6 +256,17 @@ def post(self):
         return time_entries_dao.create(ns.payload), HTTPStatus.CREATED
 
 
+@ns.route('/latest')
+class LatestTimeEntries(Resource):
+    @ns.doc('list_latest_time_entries')
+    @ns.marshal_list_with(time_entry)
+    @ns.response(HTTPStatus.NOT_FOUND, 'Time entry not found')
```

Comment:
A better message would be:
```
No time entries found.
```
It is not returning one by many, right?

Reply:
The error message is changed already